pkg/cmd/cli/cmd/importimage.go

```diff
@@ -26,12 +26,21 @@ import (
 
 var (
 	importImageLong = templates.LongDesc(`
-		Import tag and image information from an external Docker image repository
+		Import the latest images from a Docker registry into an image stream
```

Comment:
I'm trying to stay away from saying that we import images to IS. This creates a lot of confusion for our users, that's why it explicitly said image information before and I'd like to keep it that way.